apps/web/src/components/layout/components/v2/HeaderNav.tsx

```diff
@@ -33,6 +38,12 @@ export function HeaderNav() {
     >
       {/* TODO: Change position: right to space-between for breadcrumbs */}
       <HStack flexWrap={'nowrap'} justifyContent="flex-end" gap={'100'}>
+        {/* TODO: Use a more robust / realistic way to conditionally show this button */}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ From my understanding, this button should show up on all environments (Local, Dev, Prod) once v2 is released. I would imagine hiding it behind a feature flag would be the correct choice no?

Reply:
Yes @denis-kralj-novu great catch! Joel is out today, do you mind adding to this PR so we can merge it?

Reply:
Sure. I've already used the flag in my PR that is handling the logic for the top buttons so I know what to do

Reply:
Done 🚀